com/williamfiset/algorithms/sorting/ShellSort.java

```diff
@@ -0,0 +1,60 @@
+// Java implementation of ShellSort 
+class ShellSort 
+{ 
+	/* An utility function to print array of size n*/
+	static void printArray(int arr[]) 
```

Comment:
You can remove this and use java.util.Arrays.toString to print a primitive array

Reply:
ok, then what else i should do , if you can suggest me ?

Reply:
In main instead of:

System.out.println("Array before sorting"); 
printArray(arr); 

do:

System.out.println("Array before sorting: " + java.util.Arrays.toString(arr));